entgql/template.go

```diff
@@ -261,3 +264,14 @@ func removeOldTemplate(g *gen.Graph, name string) error {
 	}
 	return nil
 }
+
+// annotation extracts the entgql.Annotation or returns its empty value.
+func annotation(ants gen.Annotations) (*Annotation, error) {
```

Comment:
please use `decodeAnnotation` function,

Reply:
decode feels to me like an internal detail. From the caller's perspective, I'm interested to get the `entgql.Annotation` and don't care how it's done.

Reply:
That function is existes in the codebase, so re-use it or rename it, please.

Reply:
oh, got it. I didn't see the `decodeAnnotation` function. Thanks